### ISSUE TITLE
I'm very interested in this project!

### ISSUE THREAD

**Issue by MatthewMerritt** [pauljohanneskraft/Map#8]
I'm having trouble getting a demo app to compile and run. Would you be willing to create a very simple example app to get a new users for your package up and running?

**Comment by pauljohanneskraft** [comment]
Could you please provide some more information about this?
- Which Xcode version are you running?
- Which error(s) are visible in Xcode when trying to build?
- What is the platform you are trying to compile your example app for?
- How did you link this library using Swift Package Manager? This project is currently not available on Cocoapods and if you have not previously used Swift Package Manager, this might also be the issue here.

**Comment by pauljohanneskraft** [comment]
As an example, you can have a look at [this project](https://github.com/pauljohanneskraft/AISVisualizer) of mine that works on macOS and iOS. It uses this library for the `VisualizationPage` view.

**Comment by MatthewMerritt** [comment]
At first I was not importing MapKit, which obviously wasn't finding any of the associated types. So when I tried to import MapKit it would give the compiler a segmentation fault. I'm using the latest Xcode (Version 13.3.1 (13E500a)), with an empty created Project for iOS. Then adding Map with SPM. I loaded your test project ( thank you by the way for sharing ), with the same result. I quit and restarted Xcode and everything is working as expected. My first project worked as expected after the restart. I'm sorry I bothered you with such a trivial task, I should have tried that first... threw me though since I had other projects open that were still able to compile and if I commented out the Map(...) in my original project it compiled.
